packages/contracts/contracts/optimistic-ethereum/libraries/utils/Lib_Pausable.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+
+/**
+ * @title Pausable
+ * @dev Adapted from https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/security/Pausable.sol
```

Comment:
Everything below this line is _identical_ to [Pausable](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/security/Pausable.sol), except for replacing `Context._msgSender()` with just `msg.sender`